File: sdk/async-scala/src/main/scala/io/swagger/client/model/LineItem.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class LineItem (
  ID: Option[String],
ProductID: Option[String],
Quantity: Option[Integer],
UnitPrice: Option[Number],
CostCenter: Option[String],
DateNeeded: Option[Date],
ShippingAccount: Option[String],
ShippingAddressID: Option[String],
ShipfromAddressID: Option[String],
ShipperID: Option[String],
Specs: Option[List[ShipmentItem]],
xp: Option[Any]
)

File: sdk/slim/lib/Models/LineItem.php
<?php
/*
 * LineItem
 */
namespace \Models;

/*
 * LineItem
 */
class LineItem {
    /* @var string $iD  */
    private $iD;
/* @var string $productID  */
    private $productID;
/* @var int $quantity  */
    private $quantity;
/* @var Number $unitPrice  */
    private $unitPrice;
/* @var string $costCenter  */
    private $costCenter;
/* @var \DateTime $dateNeeded  */
    private $dateNeeded;
/* @var string $shippingAccount  */
    private $shippingAccount;
/* @var string $shippingAddressID  */
    private $shippingAddressID;
/* @var string $shipfromAddressID  */
    private $shipfromAddressID;
/* @var string $shipperID  */
    private $shipperID;
/* @var \\Models\ShipmentItem[] $specs  */
    private $specs;
/* @var object $xp  */
    private $xp;
}
